api/v1/region_types.go

```diff
@@ -15,14 +15,33 @@ const (
 type QdrantCloudRegionSpec struct {
 	// Id specifies the unique identifier of the region
 	Id string `json:"id,omitempty"`
+	// Components specifies the list of components to be installed in the region
+	// +optional
+	Components []ComponentReference `json:"components,omitempty"`
 	// HelmRepositories specifies the list of helm repositories to be created to the region
+	// Deprecated: Use "Components" instead
 	// +optional
 	HelmRepositories []HelmRepository `json:"helmRepositories,omitempty"`
 	// HelmReleases specifies the list of helm releases to be created to the region
+	// Deprecated: Use "Components" instead
 	// +optional
 	HelmReleases []HelmRelease `json:"helmReleases,omitempty"`
 }
 
+type ComponentReference struct {
+	// APIGroup is the group of the component being referenced.
+	// +optional
+	APIGroup string `json:"apiGroup,omitempty"`
+	// Kind is the type of component being referenced
+	Kind string `json:"kind"`
+	// Name is the name of component being referenced
+	Name string `json:"name"`
+	// Namespace is the namespace of component being referenced
+	// +kubebuilder:default="default"
```

Comment:
are we gonna have always `default` namespace? Should not be better that the default value is something else?

Reply:
We can default to `qdrant` namespace but we can't guarantee that this namespace will exist.

Reply:
Made namespace required.